tools/src/test/resources/basic.adl

```diff
@@ -1,4 +1,4 @@
-archetype (adl_version=2.0.5; rm_release=1.0.2; generated; controlled; build_uid=B430138C-1DD2-42EF-B54C-633909437054)
+archetype (adl_version=2.0.5-alpha; rm_release=1.0.2; generated; controlled; uid=a2bc5e00-c67e-4d7e-bb87-b3b74cdefd00; build_uid=B430138C-1DD2-42EF-B54C-633909437054; adl_version=1.0;rm_release= 2;generated= 3;controlled = 4.4.4;uid=5;build_uid=6.0.0;identifier=a2bc5e00-c67e-4d7e-bb87-b3b74cdefd00)
```

Comment:
nice test case! If you add an unknown identifier, for example `;created_by_eline`, it will cover the last line missing coverage as well.

Reply:
Poah... I am not worthy 😜 I thought of something even more creative!